src/controller/CommissioningDelegate.h

```diff
@@ -507,6 +510,41 @@ class CommissioningParameters
         return *this;
     }
 
+    Optional<NodeId> GetICDControllerNodeId() const { return mICDControllerNodeId; }
```

Comment:
Is there a reason the naming here does not match the spec (CheckInNodeID)?  It took me a while to figure out what "controller node id" was, exactly.

Reply:
done